Formula/onedrive.rb

```diff
@@ -0,0 +1,25 @@
+class Onedrive < Formula
```

Comment:
`OnedriveClient` or similar seems more appropriate. This will conflict with `homebrew/cask/onedrive` otherwise.